pulp_rpm/app/models/package.py

```diff
@@ -219,6 +223,9 @@ class Meta:
             'name', 'epoch', 'version', 'release', 'arch', 'checksum_type', 'pkgId'
         )
 
+    class ReadonlyMeta:
+        readonly = ["evr"]
```

Comment:
This is necessary because otherwise Django will attempt to insert an empty string into the field by default.

This requires the ```django-readonly-field``` extension which has to be listed in ```INSTALLED_APPS``` in ```settings.py``` -- so possibly a Pulpcore change.